package.json

```diff
@@ -134,7 +134,7 @@
     "escape-html": "^1.0.3",
     "eventemitter3": "^4.0.7",
     "formik": "^2.0.8",
-    "framer-motion": "^1.6.15",
+    "framer-motion": "^4.1.17",
```

Comment:
Have you checked that everything is working correctly after this update?

I had to tweak a few animation configs after doing a similar update in Embark I remember...